### PR TITLE
Add Estimator Python API and Inception Example

### PR DESCRIPTION
**Pull request by cyita**

### CODE REVIEW

File: pyzoo/zoo/examples/inception/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+--learningRate 0.065 \
+--weightDecay 0.0002 \
+--checkpointIteration 1000 \ 
+-f hdfs://... \
+--checkpoint /models/inception \ 
+--maxIteration 90000
+```
+
+* Spark yarn client mode, example command
+```
```
Comment (qiuxin2012) [inline pyzoo/zoo/examples/inception/README.md]: bash

File: pyzoo/zoo/feature/common.py [hunk 2]
```diff
@@ -246,7 +246,7 @@ def image_frame(cls, image_frame, memory_type="DRAM", bigdl_type="float"):
     def image_set(cls, imageset, memory_type="DRAM", bigdl_type="float"):
         """
         Create FeatureSet from ImageFrame.
-        :param image_frame: ImageFrame
+        :param imageset: ImageFrame
```
Comment (qiuxin2012) [inline pyzoo/zoo/feature/common.py]: `ImageFrame` to ImageSet

File: pyzoo/zoo/feature/common.py [hunk 3]
```diff
@@ -257,6 +257,20 @@ def image_set(cls, imageset, memory_type="DRAM", bigdl_type="float"):
...
         return cls(jvalue=jvalue)
 
+    @classmethod
+    def sample_rdd(cls, rdd, memory_type="DRAM", bigdl_type="float"):
+        """
+        Create FeatureSet from RDD[Sample].
+        :param rdd: A RDD[Sample]
+        :param memory_type: string, DRAM or PMEM
+                            If it's DRAM, will cache dataset into dynamic random-access memory
+                            If it's PMEM, will cache dataset into Intel Optane DC Persistent Memory
```
Comment (qiuxin2012) [inline pyzoo/zoo/feature/common.py]: Append following comment like `image_frame`
``` If it's a Int number n, will cache dataset into disk, and only hold 1/n
                              of the data into memory during the training. After going through the
                              1/n, we will release the current cache, and load another 1/n into
                              memory.```

File: pyzoo/zoo/feature/common.py [hunk 4]
```diff
@@ -246,7 +246,7 @@ def image_frame(cls, image_frame, memory_type="DRAM", bigdl_type="float"):
     def image_set(cls, imageset, memory_type="DRAM", bigdl_type="float"):
         """
         Create FeatureSet from ImageFrame.
-        :param image_frame: ImageFrame
+        :param imageset: ImageFrame
         :param memory_type: string, DRAM or PMEM
                             If it's DRAM, will cache dataset into dynamic random-access memory
                             If it's PMEM, will cache dataset into Intel Optane DC Persistent Memory
```
Comment (qiuxin2012) [inline pyzoo/zoo/feature/common.py]: comment here

File: pyzoo/zoo/feature/image/imagePreprocessing.py [hunk 5]
```diff
@@ -339,6 +355,14 @@ def __init__(self, bigdl_type="float"):
         super(ImageFeatureToTensor, self).__init__(bigdl_type)
 
 
+class ImageFeatureToSample(Preprocessing):
+    """
+    a Transformer that get Sample from ImageFeature.
```
Comment (qiuxin2012) [inline pyzoo/zoo/feature/image/imagePreprocessing.py]: A transformer

File: zoo/src/main/scala/com/intel/analytics/zoo/feature/image/ImageMirror.scala [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.zoo.feature.image
+
+import com.intel.analytics.bigdl.transform.vision.image.opencv.OpenCVMat
+import com.intel.analytics.bigdl.transform.vision.image.{FeatureTransformer, ImageFeature}
+import org.opencv.core.Core
+
+class Mirror() extends FeatureTransformer {
```
Comment (qiuxin2012) [inline zoo/src/main/scala/com/intel/analytics/zoo/feature/image/ImageMirror.scala]: Please add comment for all public class.

**Comment by qiuxin2012** [comment]
Is jenkins passed？

**Comment by cyita** [comment]
> Is jenkins passed？

Passed. http://10.239.47.211:18888/view/ZOO-PR-PIPELINE/job/ZOO-PR-Validation/2601/

**Pull request merged**